examples/vue-instantsearch/src/App.vue

```diff
@@ -97,11 +97,20 @@ export default {
       // prefix = page (0-based) * hitsPerPage
       // position is 1-based
       const position = this.prefix + index + 1;
-      window.aa("click", { objectID, position });
+      window.aa("click", {
+        eventName: "hit-clicked",
+        indexName: process.env.INDEX_NAME,
```

Comment:
We should document the "best" way to access all those information.

Reply:
issue filed [here](https://github.com/algolia/search-insights.js/issues/25)